recipe/build.sh

```diff
@@ -17,15 +20,14 @@ fi
 mkdir -p build
 cd build
 
-cmake \
+cmake ${CMAKE_ARGS} \
   -DCMAKE_INSTALL_PREFIX="${PREFIX}" \
   -DCMAKE_BUILD_TYPE=Release \
   -DLLVM_BUILD_LLVM_DYLIB=ON \
   -DLLVM_LINK_LLVM_DYLIB=ON \
   -DLLVM_BUILD_TOOLS=ON \
   -DLLVM_BUILD_UTILS=ON \
-  -DMLIR_ENABLE_BINDINGS_PYTHON=ON \
-  -DPython3_EXECUTABLE="$PYTHON" \
+  -DMLIR_ENABLE_BINDINGS_PYTHON:bool=ON \
```

Comment:
One we need to keep is this one
```suggestion
  -DMLIR_ENABLE_BINDINGS_PYTHON:bool=ON \
  -DPython3_EXECUTABLE=$PREFIX \
```

Reply:
Done in https://github.com/conda-forge/mlir-python-bindings-feedstock/pull/32/commits/dc187321817b95d3c5bb2cec03c93b1f413f140f.

Reply:
Sorry, I was typing only half-awake, it should have stayed "$PYTHON", not $PREFIX.